bundles/org.openhab.binding.modbus.helioseasycontrols/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,690 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="modbus"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="helios-ventilation-easycontrols">
+
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="tcp"/>
+		</supported-bridge-type-refs>
+
+		<label>Helios Ventilation</label>
+		<description>Supports controlling a Helios Heat-Recovery Ventilation Device with easyControls using the Modbus Gateway
+			TCP</description>
+		<category>HVAC</category>
+
+		<channel-groups>
+			<channel-group id="general" typeId="general"/>
+			<channel-group id="operation" typeId="operation"/>
+			<channel-group id="unitConfig" typeId="unitConfig"/>
+			<channel-group id="profiles" typeId="profiles"/>
+			<channel-group id="humidityControl" typeId="humidityControl"/>
+			<channel-group id="co2Control" typeId="co2Control"/>
+			<channel-group id="vocControl" typeId="vocControl"/>
+		</channel-groups>
+
+		<config-description>
+			<parameter name="refreshInterval" type="integer" unit="ms" min="30000">

```

Comment:
Is 30 sec. a reasonable min value? Do faster updates harm?  You could leave it up to the user, but only prevent zero or negative numbers.

Reply:
It works fine for me and the device doesn't react particularly fast to a lot of requests (which would actually queue up due to the locking mechanism).